QUICKSTART.md

```diff
@@ -0,0 +1,351 @@
+role.rbac.authorization.k8s.io/kcp-edge-dns-florin-5c4r0a44 created
+rolebinding.rbac.authorization.k8s.io/kcp-edge-dns-florin-5c4r0a44 created
+secret/kcp-edge-syncer-florin-5c4r0a44 created
+deployment.apps/kcp-edge-syncer-florin-5c4r0a44 created
+```
+
+Check that the edge syncer pod is running:
+
+```console
+KUBECONFIG=$florin_kubeconfig kubectl get pods -A
+```
+
+which should yield something like:
+
+```console
+NAMESPACE                         NAME                                              READY   STATUS    RESTARTS   AGE
+kcp-edge-syncer-florin-5c4r0a44   kcp-edge-syncer-florin-5c4r0a44-bb8c8db4b-ng8sz   1/1     Running   0          30s
+kube-system                       coredns-565d847f94-kr2pw                          1/1     Running   0          85s
+kube-system                       coredns-565d847f94-rj4s8                          1/1     Running   0          85s
+kube-system                       etcd-florin-control-plane                         1/1     Running   0          99s
+kube-system                       kindnet-l26qt                                     1/1     Running   0          85s
+kube-system                       kube-apiserver-florin-control-plane               1/1     Running   0          100s
+kube-system                       kube-controller-manager-florin-control-plane      1/1     Running   0          100s
+kube-system                       kube-proxy-qzhx6                                  1/1     Running   0          85s
+kube-system                       kube-scheduler-florin-control-plane               1/1     Running   0          99s
+local-path-storage                local-path-provisioner-684f458cdd-75wv8           1/1     Running   0          85s
+``` 
+
+### c. Create the nginx workload and deploy it to the florin cluster
+
+Create the `EdgePlacement` object for your workload. Its “where predicate” (the locationSelectors array) has one label selector that matches the Location object (`florin`) created earlier, thus directing the workload to `florin` edge cluster.
+
+In the `wmw-1` workspace create the following `EdgePlacement` object: 
+  
+```console
+kubectl ws root:my-org:wmw-1
+```
+
+```console
+  kubectl apply -f - <<EOF
+  apiVersion: edge.kcp.io/v1alpha1
+  kind: EdgePlacement
+  metadata:
+    name: edge-placement-c
+  spec:
+    locationSelectors:
+    - matchLabels: {"env":"prod"}
+    namespaceSelector:
+      matchLabels: {"common":"si"}
+    nonNamespacedObjects:
+    - apiGroup: apis.kcp.io
+      resources: [ "apibindings" ]
+      resourceNames: [ "bind-kube" ]
+    upsync:
+    - apiGroup: "group1.test"
+      resources: ["sprockets", "flanges"]
+      namespaces: ["orbital"]
+      names: ["george", "cosmo"]
+    - apiGroup: "group2.test"
+      resources: ["cogs"]
+      names: ["william"]
+  EOF
+```
+
+Deploy the nginx workload. Note the namespace label matches the label in the namespaceSelector for the EdgePlacement (`edge-placement-c`) object created above. 
```

Comment:
I think "deploy" is not the best word to use here.  It calls to mind other things in view, such as what the syncer does, and what the placement translator and syncer do together.  This step is more limited.  I usually say things like "describe the workload" or "create the workload prescription".